### PR TITLE
Add missin mobile strings

### PR DESCRIPTION
**Pull request by TRtomasz**

### CODE REVIEW

File: ar-AR.json [hunk 1]
```diff
@@ -399,6 +399,43 @@
...
+	"MOBILE_SERVER_OFFLINE": "Offline",
+	"MOBILE_QR_EXPIRES": "QR Code expires in ",
+	"MOBILE_LOG_INTO": "Log in to your Stremio account ",
+	"MOBILE_SCAN_QR": "Scan QR code above or go to ",
+	"MOBILE_LINK_ACCOUNT": "Link Account",
+	"MOBILE_SPEEDPICKER_CANCEL": "Cancel",
+	"MOBILE_SPEEDPICKER_LABEL": "Select Playback Speed",
+	"MOBILE_EMPTY_LIBRARY": "Your library is empty",
+	"MOBILE_NO_ADDON_ACTIVE": "No Stremio add-on active for this type.",
+	"MOBILE_ORGANIZER_WARNING": "The Stremio Organizer app provides a limited functionality. For a list of playable streams and the full Stremio experience, use Stremio on desktop.",
```
Comment (jaruba) [inline ar-AR.json:426]: "provides a limited" -> "provides limited"

File: ar-AR.json [hunk 2]
```diff
@@ -399,6 +399,43 @@
...
+	"MOBILE_SERVER_ONLINE": "Online",
+	"MOBILE_SERVER_OFFLINE": "Offline",
+	"MOBILE_QR_EXPIRES": "QR Code expires in ",
+	"MOBILE_LOG_INTO": "Log in to your Stremio account ",
+	"MOBILE_SCAN_QR": "Scan QR code above or go to ",
+	"MOBILE_LINK_ACCOUNT": "Link Account",
+	"MOBILE_SPEEDPICKER_CANCEL": "Cancel",
+	"MOBILE_SPEEDPICKER_LABEL": "Select Playback Speed",
+	"MOBILE_EMPTY_LIBRARY": "Your library is empty",
+	"MOBILE_NO_ADDON_ACTIVE": "No Stremio add-on active for this type.",
```
Comment (jaruba) [inline ar-AR.json:425]: "add-on" -> "addon"

File: ar-AR.json [hunk 3]
```diff
@@ -399,6 +399,43 @@
...
+	"MOBILE_ORGANIZER_WARNING": "The Stremio Organizer app provides a limited functionality. For a list of playable streams and the full Stremio experience, use Stremio on desktop.",
+	"MOBILE_ADDON_LOADING": "is still loading",
+	"MOBILE_ADDONS_LOADING": "add-ons are still loading",
```
Comment (jaruba) [inline ar-AR.json:428]: "add-ons" -> "addons"

Comment (TRtomasz) [reply ar-AR.json]: fixed

File: ar-AR.json (hunk 2 above)
Comment (TRtomasz) [reply ar-AR.json:425]: fixed

File: ar-AR.json [hunk 4]
```diff
@@ -399,6 +399,43 @@
...
+	"MOBILE_SPEEDPICKER_LABEL": "Select Playback Speed",
+	"MOBILE_EMPTY_LIBRARY": "Your library is empty",
+	"MOBILE_NO_ADDON_ACTIVE": "No Stremio addon active for this type.",
+	"MOBILE_ORGANIZER_WARNING": "The Stremio Organizer app provides limited functionality. For a list of playable streams and the full Stremio experience, use Stremio on desktop.",
+	"MOBILE_ADDON_LOADING": "is still loading",
+	"MOBILE_ADDONS_LOADING": "addons are still loading",
+	"MOBILE_ADDON": "Addon",
+	"MOBILE_CONFIGURE_ADDON": "Configure your addon",
+	"MOBILE_CONFIGURE_ADDON_LABEL": "Configure Addon",
+	"MOBILE_ORGANIZER_ADDONS_WARNING": "The Stremio Organizer app provides limited selection of addons. For the full Stremio experience, use Stremio on desktop.",
```
Comment (jaruba) [inline ar-AR.json:432]: this was correct as "provides a limited", only the `MOBILE_ORGANIZER_WARNING` should of been changed to "provides limited"

**Review by jaruba — APPROVED** [review]

**Pull request merged**